packages/design-system/src/styles/components/_Choice.scss

```diff
@@ -1,240 +1,224 @@
-    top: 0;
-    width: $choice-size;
+$ds-c-inset-border-width: $spacer-half;
 
-    @media (-ms-high-contrast: active), (forced-colors: active) {
-      // background-color: window;
-      border-color: ButtonText;
-    }
-  }
+// Layout styles for wrapping choice container
+.ds-c-choice-wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 0.5rem;
+  margin: 0.5rem 0;
+  max-width: $input-max-width;
+}
 
-  // Checkmark icon
-  + label::after {
-    background: transparent;
-    border: solid;
-    border-color: $color-white;
-    border-top-color: transparent;
-    border-width: 0 0 4px 4px;
-    box-sizing: content-box;
-    content: '';
-    height: 4px;
-    left: 9px;
-    opacity: 0;
-    position: absolute;
-    top: 11px;
-    transform: rotate(-45deg);
-    width: 10px;
-  }
+// Styles for label
+.ds-c-choice + label,
+.ds-c-choice + .ds-c-label {
+  cursor: pointer;
+  flex: 1 1 min-content;
+  font-weight: $font-normal;
+  margin-top: 0;
+  max-width: max-content;
+  transform: translateY(0.25em);
+}
 
-  // Choice radio button
-  &[type='radio'] + label::before {
-    border-radius: 100%;
+.ds-c-choice--small + label,
+.ds-c-choice--small + .ds-c-label {
+  transform: translateY(-0.125em);
+}
 
-    @media (-ms-high-contrast: active), (forced-colors: active) {
-      -ms-high-contrast-adjust: none;
-      background-color: window;
-    }
+// Disabled label styles
+label,
+.ds-c-label {
+  .ds-c-choice:disabled + & {
+    color: $color-muted;
   }
 
-  &[type='radio'] + label::after {
-    background-color: $choice-checked-border-color;
-    border: 0;
-    border-radius: 100%;
-    height: $choice-radio-button-size;
-    left: 5px;
-    top: 5px;
-    transform: none;
-    width: $choice-radio-button-size;
+  .ds-c-choice.ds-c-choice--inverse:disabled + & {
+    color: $color-muted-inverse;
+  }
 
+  .ds-c-choice:disabled + &,
+  .ds-c-choice.ds-c-choice--inverse:disabled + & {
+    cursor: not-allowed;
+    
     @media (-ms-high-contrast: active), (forced-colors: active) {
-      -ms-high-contrast-adjust: none;
-      background-color: WindowText;
+      color: GrayText;
     }
   }
+}
 
-  &:checked[type='radio'] + label::before {
-    background-color: $color-white;
-
-    @media (-ms-high-contrast: active), (forced-colors: active) {
-      background-color: window;
-    }
+// Styles inherited by both checkbox and radio
+.ds-c-choice {
+  align-items: center;
+  appearance: none;
+  background-color: $choice-background-color;
+  border: $choice-border-width solid $choice-border-color;
+  border-radius: $choice-border-radius;
+  cursor: pointer;
+  display: grid;
+  height: $choice-size;
+  justify-items: center;
+  margin: 0;
+  place-content: center;
+  position: relative;
+  width: $choice-size;
+
+  &::before {
+    box-shadow: inset 0 0 4em 1em $choice-checked-background-color;
+    content: '';
+    grid-area: 1/-1;
+    height: $choice-size;
+    opacity: 0;
+    transition: 120ms opacity ease-in-out;
```

Comment:
@line47 @ericwbailey @babsmiles This is the only visual change I implemented - it's a subtle animation when users select/deselect a choice. Lemme know if yinz think it's gross.

Reply:
I think this looks good and also have 2 thoughts on adding in this animation...


1. Should we use some animation settings from our variables? https://github.com/CMSgov/design-system/blob/master/packages/design-system/src/styles/settings/variables/_animation.scss you can see a commented out example here. https://github.com/CMSgov/design-system/blob/7a0f6c1c2bfcc74b773f14bf63c156972a232a8c/packages/design-system/src/styles/components/_Tabs.scss#L44

2. My second thought is we've never established animation guidelines or a plan for how animation would be integrated into the design system which makes me think...should we establish a deliberate direction for our animations? This makes me want to take a step back and have a more defined approach to animation and how it could and would be applied to components that exists in the design system today.

Reply:
Good points. I can refactor the transition I currently have to use one of our pre-existing variables (or comment it out). 

I think #2 would be a worthwhile (and fun) exercise. I imagine there's a few components that could benefit from animation.

Reply:
I agree on the second point and think it could be a nice touch to the system overall, thanks for bringing this up! Should we backlog an animation plan?

Reply:
I just thought of another consideration: I'm not sure how this change would affect teams' visual regression tests that include this component in a checked state. Would their testing suite know to wait for the animation end to snap the picture? If it's a race, how likely is it to get non-deterministic image results?

Reply:
I do like the transition visually

Reply:
@line47 I tried using `$animation-speed-1` in this transition, but it was slower than the 120ms I currently have and made it look a little laggy. I'm going to comment it out until we have a better animation guidelines in place.

Reply:
Sounds good, thanks for testing!